resource/composed/composed_test.go

```diff
@@ -65,6 +65,8 @@ func ExampleScheme() {
 	// Add all v1beta2 types to the scheme so that From can automatically
 	// determine their apiVersion and kind.
 	v1beta2.AddToScheme(Scheme)
+
+	// Output:
```

Comment:
is this leftover from operations work ongoing at the same time?

Reply:
No, a linter complained that the example needs an `Output` block to be valid for `go test`, which can test examples. In this case it expects no output.